hello-books-swagger.json

```diff
@@ -250,6 +287,34 @@
       }
     }
   },
+  "/books": {
```

Comment:
Hi, @chiboycalix ,nice work.  The books route here does not display on the documentation. You may want to check and fix that.

Reply:
I have made the adjustments.

Reply:
@chiboycalix Nice work, bro. But somehow the books route doesn't work. You may also want to look into the way you defined your schema. The problem could be from there.